region/lowernorfair/west.json

```diff
@@ -782,11 +806,68 @@
                   "canHeatRun"
                 ]},
                 {"or": [
+                  {"obstacle":{
+                    "id": "A",
+                    "requires": [
+                      "canIBJ",
+                      {"cost": [
+                        {"heatFrames": 2000}
+                      ]}
+                    ]
+                  }},
                   {"and": [
-                    "ScrewAttack",
                     "SpaceJump",
                     {"cost": [
                       {"heatFrames": 400}
+                    ]},
+                    {"obstacle":{
+                      "id": "A",
+                      "requires": [
+                        {"or":[
+                          "ScrewAttack",
+                          {"and":[
+                            "canUsePowerBombs",
+                            {"cost": [
+                              {"heatFrames": 150},
+                              {"ammo": [
+                                {
+                                  "type": "PowerBomb",
+                                  "count": 1
+                                }
+                              ]}
+                            ]}    
+                          ]}
+                        ]}
+                      ]
+                    }}
+                  ]},
+                  {"and": [
+                    {"cost": [
+                      {"heatFrames": 400}
+                    ]},
+                    {"obstacle":{
+                      "id": "A",
+                      "requires": [
+                        {"and":[
+                          "canUsePowerBombs",
+                          {"cost": [
+                            {"heatFrames": 150},
+                            {"ammo": [
+                              {
+                                "type": "PowerBomb",
+                                "count": 1
+                              }
+                            ]}
+                          ]}    
+                        ]}
+                      ]
+                    }},
+                    {"or": [
+                      "canSpringBallWalljumpMidAir",
+                      {"and": [
+                        "HiJump",
+                        "SpeedBooster"
```

Comment:
If Obstacle 'B' is broken this needs a reset

Reply:
Nah, the setup here is inside the doorway of node 2 so not reliant on obstacle B.